core/src/main/scala/org/scalafmt/internal/Router.scala

```diff
@@ -409,6 +409,52 @@ class Router(formatOps: FormatOps) {
                 .withIndent(2, expire, Left)
             )
         }
+
+      // Parameter opening for one parameter group. This format works
+      // on a group-by-group basis.
```

Comment:
This doesn't sound right to me. I would like to have a holistic approach where I see the `def` as one thing, not as a bunch of `(`. I thought I could do so by using the `Decision` below, but for some reason a Decision for `(` (same as the top format case) just isn't picked up. 

Is there another way of doing this?